src/routes/members/[studentId]/UpdateMemberForm.svelte

```diff
@@ -60,12 +65,17 @@
         {/each}
       </select>
     </Labeled>
-    <Labeled label="Year" id="classProgramme" error={$errors.classYear}>
+    <Labeled
+      label="Year"
+      id="classProgramme"
+      error={$errors.classYear}
+      fullWidth={true}
+    >
       <input
         type="number"
         name="classYear"
         id="classYear"
-        class="input input-bordered"
+        class="w-xs input input-bordered"
```

Comment:
`w-xs` can be replaced by a Tailwind class:
```suggestion
        class="input input-bordered md:max-w-20"
```

Reply:
Yes, that's what I wanted to do but was met with this error: ```Attributes should not contain ':' characters to prevent ambiguity with Svelte directives(illegal-attribute-character) ```

Reply:
That's odd, it should definitely work.

Reply:
In that case, should I simply commit your suggestions?